instances/terminology/annotationType.jsonld

```diff
@@ -4,7 +4,7 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/terminology/annotationType",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Terminology",
-  "definition": "General classification of how data were annotated.",
+  "definition": "General classification of annotations into geometry types.",
```

Comment:
Geometrical classification of annotations into types.

Reply:
Sounds good. I changed it.